component/file_cache/file_cache_test.go

```diff
@@ -1413,9 +1413,9 @@ func (suite *fileCacheTestSuite) TestCachePathSymlink() {
 	suite.assert.EqualValues(data, d)
 }
 
-func (suite *fileCacheTestSuite) TestZZDirectRead() {
+func (suite *fileCacheTestSuite) TestZZOffloadIO() {
 	defer suite.cleanupTest()
-	configuration := fmt.Sprintf("file_cache:\n  path: %s\n  direct-read: true\n  timeout: 0\n\nloopbackfs:\n  path: %s",
+	configuration := fmt.Sprintf("file_cache:\n  path: %s\n  timeout: 0\n\nloopbackfs:\n  path: %s",
```

Comment:
What is this test testing now?

Reply:
direct-read has been replaced with offload-io and default is to use native code (= direct-read) so test case remains the same but this config is not needed anymore.